### PR TITLE
update system cube

### PR DESCRIPTION
**Pull request by xuekaiqi**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Kylin?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have create an issue on [Kylin's jira](https://issues.apache.org/jira/browse/KYLIN), and have described the bug/feature there in detail
- [ ] Commit messages in my PR start with the related jira ID, like "KYLIN-0000 Make Kylin project open-source"
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If this change need a document change, I will prepare another pr against the `document` branch
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at user@kylin or dev@kylin by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: website/_docs/tutorial/setup_systemcube.cn.md [hunk 1]
```diff
@@ -356,17 +376,25 @@ sh ${KYLIN_HOME}/bin/kylin.sh org.apache.kylin.tool.job.CubeBuildingCLI --cube $
   <tr>
     <th colspan="2">Dimension</th>
   </tr>
+  <tr>
+    <td>HOST</td>
+    <td>the host of server for query engine</td>
```
Comment (hit-lacus) [inline website/_docs/tutorial/setup_systemcube.cn.md:381]: Should be `job engine` or `query engine`?

File: website/_docs/tutorial/setup_systemcube.cn.md [hunk 2]
```diff
@@ -7,16 +7,24 @@ permalink: /cn/docs/tutorial/setup_systemcube.html
...
+## <span id="什么是系统 Cube">什么是系统 Cube</span>
 
 为了更好的支持自我监控，在系统 project 下创建一组系统 Cubes，叫做 "KYLIN_SYSTEM"。现在，这里有五个 Cubes。三个用于查询指标，"METRICS_QUERY"，"METRICS_QUERY_CUBE"，"METRICS_QUERY_RPC"。另外两个是 job 指标，"METRICS_JOB"，"METRICS_JOB_EXCEPTION"。
 
-## 如何建立系统 Cube
+## <span id="如何建立系统 Cube">如何建立系统 Cube</span>
 
-### 准备
-在 KYLIN_HOME 目录下创建一个配置文件 SCSinkTools.json。
+本节我们介绍手动启用系统 Cube 的方法，如果您希望通过 shell 脚本自动创建系统 Cube，请参考[自动创建系统 Cube](#什么是系统 Cube)。
```
Comment (hit-lacus) [inline website/_docs/tutorial/setup_systemcube.cn.md:23]: Is the link `(#什么是系统 Cube)` right ?

File: website/_docs/tutorial/setup_systemcube.md [hunk 3]
```diff
@@ -383,56 +414,82 @@ This Cube is for collecting job metrics. The details are as follows:
...
   </tr>
   <tr>
     <td>CUBING_TYPE</td>
-    <td></td>
+    <td>in kylin，there are two cubing algorithms，Layered & Fast(InMemory)</td>
   </tr>
   <tr>
     <td>EXCEPTION</td>
-    <td>when running a job, exceptions may happen. It's for classifying different exception types</td>
+    <td>when running a job，exceptions may happen. It's for classifying different exception types</td>
```
Comment (hit-lacus) [inline website/_docs/tutorial/setup_systemcube.md:488]: Why using chinese character?

File: website/_docs/tutorial/setup_systemcube.cn.md (hunk 2 above)
Comment (xuekaiqi) [reply website/_docs/tutorial/setup_systemcube.cn.md:23]: It is wrong... and I will modify it

File: website/_docs/tutorial/setup_systemcube.md [hunk 4]
```diff
@@ -383,56 +414,82 @@ This Cube is for collecting job metrics. The details are as follows:
...
 ### METRICS_JOB_EXCEPTION
 This Cube is for collecting job exception metrics. The details are as follows:
 
 <table>
   <tr>
     <th colspan="2">Dimension</th>
   </tr>
+  <tr>
+    <td>HOST</td>
+    <td>the host of server for query engine</td>
```
Comment (hit-lacus) [inline website/_docs/tutorial/setup_systemcube.md:464]: Correct here.

File: website/_docs/tutorial/setup_systemcube.md [hunk 5]
```diff
@@ -383,56 +414,82 @@ This Cube is for collecting job metrics. The details are as follows:
...
     <td>the size of data source in bytes</td>
   </tr>
   <tr>
-    <td>MIN, MAX, SUM of CUBE_SIZE</td>
+    <td>MIN，MAX，SUM of CUBE_SIZE</td>
     <td>the size of created Cube segment in bytes</td>
   </tr>
   <tr>
-    <td>MIN, MAX, SUM of PER_BYTES_TIME_COST</td>
+    <td>MIN，MAX，SUM of PER_BYTES_TIME_COST</td>
```
Comment (hit-lacus) [inline website/_docs/tutorial/setup_systemcube.md:429]: Not use chinese character please.

File: website/_docs/tutorial/setup_systemcube.md [hunk 6]
```diff
@@ -189,19 +202,19 @@ This Cube is for collecting query metrics at the highest level. The details are
     <td></td>
   </tr>
   <tr>
-    <td>MIN, MAX, SUM of QUERY_TIME_COST</td>
+    <td>MIN，MAX，SUM，PERCENTILE_APPROX of QUERY_TIME_COST</td>
```
Comment (hit-lacus) [inline website/_docs/tutorial/setup_systemcube.md:205]: Is it correct?

Comment (hit-lacus) [reply website/_docs/tutorial/setup_systemcube.md:205]: `，` or `,`

**Review by hit-lacus — APPROVED** [review]
Approved

**Pull request merged**